Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TurkishAirLinesHelper.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/19/17.
+//  Copyright © 2017 Modeso. All rights reserved.
+//
+
+import Foundation
+
+class TurkishAirLinesHelper {
+    
+    enum GameDataStatus {
+        case schedule
+        case results
+    }
+    
+    let urls: Dictionary<GameDataStatus, String> = [
```

Comment:
Also we can set the base url in Almofire one time and after that just pass to it the remaining path like "chedules?seasoncode=E2016"